frontend/app/src/components/settings/OracleEntry.vue

```diff
@@ -42,6 +48,12 @@ export default defineComponent({
         return require('@/assets/images/oracles/coingecko.svg');
       } else if (get(identifier) === 'manual') {
         return require('@/assets/images/oracles/book.svg');
+      } else if (identifier.value === 'uniswapv2') {
+        return require('@/assets/images/defi/uniswap.svg');
+      } else if (identifier.value === 'uniswapv3') {
+        return require('@/assets/images/defi/uniswap.svg');
+      } else if (identifier.value === 'saddle') {
```

Comment:
Yes but only for one asset at the moment. I would leave it there and don't add it to the changelog since is not that useful and can be improved by a lot in the future